multidim-interop/README.md

```diff
@@ -0,0 +1,58 @@
+# Interoperability test
+
+This tests that different libp2p implementations can communicate with each other
+on each of their supported capabilites.
+
+Each version of libp2p is defined in `versions.ts`. There the version defines
+its capabilities along with the id of its container image.
+
+# Test spec
+
+The implementation is run in a container and is passed parameters via
+environment variables. The current parameters are:
+
+| Name         | Description                                                  | Is Optional                                                     |
+| ------------ | ------------------------------------------------------------ | --------------------------------------------------------------- |
+| REDIS_ADDR   | A different address to connect to redis (default redis:6379) | yes                                                             |
+| ip           | IP address to bind the listener to                           | no                                                              |
+| transport    | The transport to use                                         | no                                                              |
+| muxer        | The muxer to use                                             | no, except when transport is one of quic, quic-v1, webtransport |
+| security     | The security channel to use                                  | no, except when transport is one of quic, quic-v1, webtransport |
+| is_dialer    | Should you dial or listen                                    | no                                                              |
+| test_timeout | Control the timeout of test. In seconds                      | yes, default to 10 seconds                                      |
+
+The test should do two different things depending on if it's the dialer or
+listener.
+
+## Dialer
+
+The dialer should emit all diagnostic logs to `stderr`. Only the final JSON
+string result should be emitted to `stdout`.
+
+1. Connect to the Redis instance.
+2. Create a libp2p node as defined by the environment variables.
+3. Get the listener's address via Redis' `BLPOP` using the `listenerAddr` key.
+4. Record the current time as `handshakeStart`.
+5. Connect to the listener.
+6. Ping the listener, and record the round trip time as `pingRTT`
```

Comment:
```suggestion
6. Ping the listener, and record the duration of the round trip as `pingRoundTripDuration`
```

(idea for clarity to avoid the ambiguous term "time".)

Reply:
RTT is a well know term. I'll keep it.